stylesheets/styles.css

```diff
@@ -1,8 +1,9 @@
 @import url(https://fonts.googleapis.com/css?family=Open+Sans:400italic,400,600,300,700);
 
 :root {
-  --brand-primary: #017d87;
-  --brand-accent: #005d64;
+  --brand-primary: #00E1BB;
```

Comment:
The primary colour is no longer being used anywhere but it felt odd to remove it so I have left it for now.